tensorflow/g3doc/get_started/os_setup.md

```diff
@@ -272,6 +272,11 @@ You can find out the directory with the following command:
 $ python -c 'import os; import inspect; import tensorflow; print(os.path.dirname(inspect.getfile(tensorflow)))'
 ```
 
+python3:
+```bash
+$ python3 -c 'import os; import inspect; import tensorflow; print(os.path.dirname(inspect.getfile(tensorflow)))'
+```
+
```

Comment:
Instead of basically repeating all the code with a minor change, could we instead change line 269 and append "(make sure to use the python command you have installed TensorFlow with, for example, use `python3` instead of `python` if you installed for Python 3)"

I think that's probably more readable.